core/src/main/java/org/apache/accumulo/core/util/ratelimit/RateLimiter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.util.ratelimit;
+
+public interface RateLimiter {
+  /**
+   * Get current QPS of the rate limiter, with a nonpositive rate indicating no limit.
```

Comment:
Yes.  I chose to say QPS (queries per second) here as it is the terminology used by `com.google.common.util.concurrent.RateLimiter` to describe its functionality.  Things other than byte counts might potentially wish to be rate limited.